src/components/generic/movies/movies.tsx

```diff
@@ -43,23 +43,28 @@ const RenderMovies = (movies: Movie[], loader: boolean) => {
   return (
     <>
       {movies !== undefined &&
-        movies.map(({poster_med, rating, year, genres, title, imdb}, key) => (
-          <ModalProvider.Consumer key={key}>
-            {({handleOpen}) => {
-              return (
-                <MovieCard
-                  title={title}
-                  imdb={imdb}
-                  genres={genres}
-                  posterMed={poster_med}
-                  rating={rating}
-                  year={year}
-                  handleOpen={handleOpen}
-                />
-              )
-            }}
-          </ModalProvider.Consumer>
-        ))}
+        movies.map(
```

Comment:
```suggestion
        movies?.map(
```

In the new JS you can now do optional chaining. Doing this means that you can remove the line above.

Reply:
Oh that's cool, I will use it. Its also supported in Dart